platinum-bluetooth-characteristic.html

```diff
@@ -126,16 +125,52 @@
      * });
      * ```
      *
+     * Starting and stopping notifications on a `<platinum-bluetooth-characteristic>` is pretty straightforward when taking advantage of [Polymer Change notification protocol](https://www.polymer-project.org/1.0/docs/devguide/data-binding.html#change-notification-protocol). Here's how to get your Heart Rate Measurement for instance:
+     *
+     * ```html
+     * <platinum-bluetooth-device services-filter='["heart_rate"]'>
+     *   <platinum-bluetooth-service service='heart_rate'>
+     *     <platinum-bluetooth-characteristic characteristic='heart_rate_measurement'>
+     *     </platinum-bluetooth-characteristic>
+     *   </platinum-bluetooth-service>
+     * </platinum-bluetooth-device>
+     * ```
+     * ```js
+     * var bluetoothDevice = document.querySelector('platinum-bluetooth-device');
+     * var heartRate = document.querySelector('platinum-bluetooth-characteristic');
+     *
+     * startButton.addEventListener('click', function() {
+     *   bluetoothDevice.request().then(function() {
+     *     return heartRate.startNotifications().then(function() {
+     *       heartRate.addEventListener('value-changed', handleValueChanged);
```

Comment:
This again could be an attribute on the `<platinum-bluetooth-characteristic>` element, to avoid the need to add and remove event listeners. You'd just call `startNotifications().catch(error=>{})`.